### ISSUE TITLE
heatmaps not overlayed on video when exporting

### ISSUE THREAD

**Issue by yemu** [pupil-labs/pupil#674]
Hi,

when I try to export video data, with heatmaps display enabled in surface tracker (instead of marker IDs), the exported video doesn't include heatmaps (heatmaps are displayed in pupil player window). 
is it possible to have them exported in the video?
y

**Comment by mkassner** [comment]
The heatmaps are exported as png's . Since they are rendered on the GPU its not trivial to also include the in the exported video.

**Comment by yemu** [comment]
yes, I know they are in the exported folder, but the effect of having heatmaps overlayed is really cool :)

**Comment by papr** [comment]
@mkassner This could be closed with a `wontfix`.
